pkg/volume/glusterfs/glusterfs.go

```diff
@@ -454,6 +540,23 @@ func (d *glusterfsVolumeDeleter) Delete() error {
 
 	glog.V(4).Infof("glusterfs: deleting volume %q with configuration %+v", volumeId, d.provisioningConfig)
 
+	gid, err := d.getGid()
```

Comment:
if deleting an old one, it won't have a gid, right? we should tolerate missing ones